packages/app-store/rainbow/README.md

```diff
@@ -0,0 +1,12 @@
+# Rainbow Plugin
+
+> Intended for developer information
+
+The web3 app uses [RainbowKit](https://www.rainbowkit.com/) to authenticate Ethereum users.
+
+When deploying, the app requires either a `ALCHEMY_API_KEY` or `INFURA_API_KEY` (or both) which can be obtained by creating an Alchemy or Infura project respectively.
+
+![Find your Alchemy API Key](https://gateway.pinata.cloud/ipfs/QmXme422MmQf1o959pLbLiJvjkyVW1cF7ALgJXphCoVKJu)
+![Find your Infura API key](https://gateway.pinata.cloud/ipfs/QmRf1jfQZdj1BpAnhazFhYroCvwPoqpKUsUpZrm4bHZRxc)
```

Comment:
what are those links? they arent loading for me

Reply:
looks like Pinata is slow/down. they're just screenshots but i'll move them to a different hosting site if it doesn't come back up